### DISCUSSION TITLE
first-sight/life/上一个“末流211”是一种什么感受—渺小而伟大

### DISCUSSION THREAD

**Discussion by giscus** [Hainanu-Application/Hainanu-Application.github.io#60] [Announcements]
# first-sight/life/上一个“末流211”是一种什么感受—渺小而伟大

A magical documentation generator.

https://hainanu-application.github.io/

**Comment by serenadeJack** [comment]
博主的经历和我很相似，目前在读大二上了，但是对自己的专业不是很满意，有机会保研，却又不甘心，还是有些迷茫。

**Comment by JonWong959** [comment]
毕业好几年了，学习氛围谈不上，生活氛围是真的好，有几个大学能在海边的操场上吹着海风开草地音乐节哈哈哈。
当年我们的信息平台只有微信的一两个公众号看看课表查查成绩，碰巧看到现在有这样的信息平台表示真的很羡慕。
